src/AssembleElemSolverAlgorithm.C

```diff
@@ -74,10 +74,8 @@ AssembleElemSolverAlgorithm::AssembleElemSolverAlgorithm(
 void
 AssembleElemSolverAlgorithm::initialize_connectivity()
 {
-  if (entityRank_ == stk::topology::ELEM_RANK)
-    eqSystem_->linsys_->buildElemToNodeGraph(partVec_);
-  else if (entityRank_ == realm_.meta_data().side_rank())
-    eqSystem_->linsys_->buildFaceToNodeGraph(partVec_);
+  eqSystem_->linsys_->buildFaceElemToNodeGraph(partVec_);
```

Comment:
Changing `initialize_connectivity` is not what I had in mind when I suggested that we uniformly call `buildElemToNodeGraph` etc.